src/lib/validation/validate.ts

```diff
@@ -74,6 +74,12 @@ export const getFormErrors = async (
   }
 };
 
+function isScalarValue(obj: IValues | Value | Values): obj is Value {
```

Comment:
Is there a reason why this is a regular function and not an arrow function adhering to the surrounding coding style?

Reply:
Yeah, had to look up exactly how to do type predicates with arrow functions and when I figured it out I found it completely unreadable.

See also https://stackoverflow.com/a/47847949

Reply:
Rewrote it as an arrow function in #29

As an aside, there are other cases where correctly typing React components with arrow functions and generics is not possible, where using a `function` declaration` is the only option. I also kind of fail the see why every function should be defined as an arrow function when plain functions work equally well and have benefits from being hoisted and being easier syntax to read, especially in combination with the syntax.